TYPO3.Neos/Resources/Private/Styles/Foundation/_utilities.scss

```diff
@@ -18,6 +18,9 @@
 .neos-show {
   display: block;
 }
+.neos-show-flex {
+  display: flex;
+}
```

Comment:
not really a good way to fix it.. instead fix it in `_modals.scss` with s.th. like

    .neos-modal-centered {
        &.neos-show {
             display: flex;
        }
    }